### PR TITLE
Add blog post and misc site updates

### PR DESCRIPTION
**Pull request by hannahfrost**
@mjgiarlo @anarchivist Can you please review this blog post? 

Also, a little detail: I'm not sure I formatted the link to the FAQ page in the post correctly. I tried to do a relative link, but that is hard to see if it will work in the preview.

### CODE REVIEW

File: _data/previous_presentations.yaml [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+- title: Building and Bundling a National Digital Platform
+  presenters:
+    - Mark Matienzo
+    - Debra Hanken Kurtz
+    - Tom Cramer
+  event: CNI Fall 2015 Membership Meeting
+  location: "Washington, DC"
+  date: 2015-12-15
+  url:  https://osf.io/h4ru8/
+  - title: Hydra in a Box
```
Comment (mjgiarlo) [inline _data/previous_presentations.yaml]: Methinks the two spaces here (before "- title") are a typo, and they break site generation:

```
$ jekyll s
Configuration file: /home/mjg/workspace/hybox.github.io/_config.yml
            Source: /home/mjg/workspace/hybox.github.io
       Destination: /home/mjg/workspace/hybox.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
jekyll 3.0.0 | Error:  (/home/mjg/workspace/hybox.github.io/_data/previous_presentations.yaml): did not find expected key while parsing a block mapping at line 1 column 3
```

File: _data/faqs.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 - question: >-
     What is the project timeline?
   answer: >-
-    The Design Phase, now underway, is scheduled to complete in March 2016. Infrastructure and technical exploration is also now underway. These two concurrent phases will converge as the requirements are finalized in winter 2016. Formal development is expected to begin in April 2016.  For more detail, see this visual timeline for August 2015 - March 2016 <img src="/images/HyBox_design_process_tasks_timeline_2015-09-18.png" width="75%"/>.
+    The Design Phase, now underway, is scheduled to complete in March 2016. Infrastructure and technical exploration is also now underway. These two concurrent phases will converge as the requirements are finalized in winter 2016. Formal development is expected to begin in April 2016.  For more detail, see this visual timeline for August 2015 - March 2016 
+    <img src="/images/HyBox_design_process_tasks_timeline_2015-09-18.png" width="75%"/>.
```
Comment (mjgiarlo) [inline _data/faqs.yaml]: This break is showing weirdly in my browser, with the image breaking up the "August 2015 - March 2016" text.

**Comment by mjgiarlo** [comment]
OK, I made [a few tweaks](https://github.com/hybox/hybox.github.io/commit/5046ea0dd6db6f741a2960d78e1a009568d7fa8a) and the site renders A-OK for me locally. I'm :+1: on the PR! Great stuff, @hannahfrost, thanks. :clap:

**Comment by mjgiarlo** [comment]
It's up to @anarchivist to :shipit: now.

**Comment by hannahfrost** [comment]
Thanks @mjgiarlo!

**Comment by anarchivist** [comment]
I, too, added a few minor adjustments in ab5a5d8. Otherwise I think this is good to go.

**Comment by hannahfrost** [comment]
I noticed that the bullet list is not displaying properly. Going to fix that.

**Comment by mjgiarlo** [comment]
:clap:

**Pull request merged**